HGustavs/LenaSYS
# testdata of listentries are created with no tabs #12639

Demo-Course and Webbutveckling - datorgrafik testdata is created witth 0 tabs instead of 2.

Testing:
1) Run install script (install/install.php)
2) Enter Demo-course or Webbutveckling - datorgrafik
3) The initial listentires should start with 0 tabs (previously started with 2 tabs).
Expected result:
![image](https://user-images.githubusercontent.com/102609651/169047038-76e7801d-a91a-4965-be1d-ed67c65f43aa.png)
